### PR TITLE
login tests | OSD-8139

### PR DESCRIPTION
**Pull request by supreeth7**

### CODE REVIEW

File: tests/login_test.go [hunk 1]
```diff
@@ -1,22 +1,66 @@
...
+				).
+				Run()
+
+			Expect(result.ConfigFile()).ToNot(BeEmpty())
+			Expect(result.ConfigString()).To(MatchJSON(jsonTemplate))
+		})
+	})
+
+	When("the login command is run", func() {
+		It("the sample key is authenticated against PagerDuty REST API", func() {
```
Comment (T0MASD) [inline tests/login_test.go:51]:
```suggestion
		It("the sample key is used against PagerDuty REST API", func() {
```

**Review by T0MASD — COMMENTED** [review]
Looks good, just a couple of minor suggestions.

File: tests/main_test.go [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+func TestPDCLI(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "PDCLI Test Suite")
+}
+
+var executable string
+
+var _ = BeforeSuite(func() {
+	binaryName := "pdcli"
```
Comment (T0MASD) [inline tests/main_test.go:26]: for consistency should this var be changed to `executableName` ?

**Comment by T0MASD** [comment]
please squash the 3 commits into 1 and it's good to go

**Comment by T0MASD** [comment]
@supreeth7 please squash the minor `tweaks commit` as it provides no additional context and can just live inside `login tests implemented`

**Comment by supreeth7** [comment]
@T0MASD can I get an lgtm?

**Comment by supreeth7** [comment]
I guess the openshift-ci bot doesn't automatically merge PRs on lgtm?

**Comment by T0MASD** [comment]
> I guess the openshift-ci bot doesn't automatically merge PRs on lgtm?

yep, please feel free to merge whenever you see green lgtm label if reviewer hasn't merged it already

**Comment by supreeth7** [comment]
@T0MASD - I don't have perms to merge PRs :)

**Pull request merged**